openwisp_controller/geo/tests/test_api.py

```diff
@@ -172,6 +174,17 @@ def test_geojson_list(self):
             self.assertContains(r, str(location_a.pk))
             self.assertContains(r, str(location_b.pk))
 
+        with self.subTest('Test filtering using organization slug'):
+            self.client.login(username='admin', password='tester')
+            response = self.client.get(
+                reverse(url), data={'organization_slug': org_a.slug}
```

Comment:
If I am not wrong, django-filters also work with query strings.
There should also be  test case for that too

Reply:
The above line of code is creating a query string.